src/pocketmine/command/CommandReader.php

```diff
@@ -40,15 +41,27 @@ public function shutdown(){
 		$this->shutdown = true;
 	}
 
+	private function initStdin(){
+		global $stdin;
+		$stdin = fopen("php://stdin", "r");
+		stream_set_blocking($stdin, 0);
+	}
+
 	private function readLine(){
 		if(!$this->readline){
 			global $stdin;
 
 			if(!is_resource($stdin)){
-				return "";
+				$this->initStdin();
 			}
-
-			return trim(fgets($stdin));
+			
+			$fgets = fgets($stdin);
+			
+			if(Utils::getOS() == "win" && $fgets === false){ //windows sucks
```

Comment:
Well, this will only fix it for windows until i find a genius way to do this. I could use some time() statements, but that doesn't sound like a good practise!